secrets-provider-chart/Chart.yaml

```diff
@@ -0,0 +1,6 @@
+apiVersion: v1
+appVersion: "1.0.0"
+description: A Helm chart for Kubernetes
+name: secrets-provider-chart
```

Comment:
Also here the name should be the official name of the product CyberArk Secrets Provider for Kubernetes

Reply:
updating the chart name to `cyberark-secrets-provider-for-kubernetes`

Reply:
this is the name they will use to install
`helm install cyberark-secrets-provider-for-kubernetes ...`